getting-started/what-you-need.md

```diff
@@ -11,197 +11,248 @@ position: 1
 
 # Typical Workflow for Using the UI for Blazor Components
 
-This article describes the steps in the typical workflow for using the Telerik UI for Blazor components&mdash;getting the Telerik UI for Blazor components and configuring your project to use them.
+This article describes the required steps to use the Telerik UI for Blazor components in any kind of Blazor application and any .NET version. The content below describes how to obtain the Telerik UI for Blazor components and configure your project to use them.
 
->tip The information in this article is also available as step-by-step tutorials for Blazor [Server]({%slug getting-started/server-side%}), [WebAssembly]({%slug getting-started/client-side%}), [.NET 8 Blazor Web App]({%slug getting-started/web-app%}) and [Blazor Hybrid]({%slug getting-started/hybrid-blazor%}) apps.
+>tip The information in this article builds on top of the step-by-step tutorials for specific Blazor application types or render modes:
+>
+> * [Blazor Web App]({%slug getting-started/web-app%}) template in .NET 8
+> * [Blazor Server apps]({%slug getting-started/server-side%}) in .NET 6 or 7
+> * [Blazor WebAssembly apps]({%slug getting-started/client-side%})
+> * [Blazor Hybrid (MAUI) apps]({%slug getting-started/hybrid-blazor%})
+>
+> The differences between the above tutorials and this page are:
+>
+> * The above tutorials are simpler to follow and are suitable for first-time Blazor or Telerik component users.
+> * The current article assumes some Blazor experience and knowledge about the Blazor application structure.
+> * The above tutorials describe the fastest and simplest way to get started with Telerik UI for Blazor.
+> * The current article provides more setup options and suggests possible enhancements.
+>
+> It's best for first-time users to start from the above tutorials and then come back to this article. Developers who have already setup Telerik Blazor apps can use directly this article.
 
-To use the Telerik UI for Blazor, you need to:
+The required milestones to obtain and use Telerik UI for Blazor are:
 
-1. Get the [Telerik Blazor packages](#telerik-specific-packages) in your project.
+1. Install the [Telerik Blazor NuGet packages](#nuget-packages) in your Blazor application.
+1. Register the [required namespaces](#namespaces).
+1. Add the [Telerik Blazor service](#service).
+1. Add the [CSS theme and JavaScript file](#css-theme-and-javascript-files).
+1. Add the [`<TelerikRootComponent>`](#telerikrootcomponent).
+1. [Add components to a view](#add-telerik-components-to-a-view)
 
-1. Add the [client assets](#adding-the-client-assets).
 
-1. [Set up the project](#configuring-the-project) to recognize the Telerik components.
+## NuGet Packages
 
+Telerik UI for Blazor is distributed through several private and public NuGet packages.
 
-## Telerik-Specific Packages
+`Telerik.UI.for.Blazor` is the only NuGet package that you must reference in the app. This package references all others as dependencies.
```

Comment:
Added a highlight to the discussed note.

All NuGet packages are listed, because there are cases when [the developer must personally take care to obtain them all and put them in a folder in their environment](https://learn.microsoft.com/en-us/nuget/hosting-packages/local-feeds).